src/translations/translations_EN.json

```diff
@@ -77,6 +77,12 @@
     "500_page": "Whoops, something went wrong. Try again later",
     "generic": "Whoops, something went wrong. Try again later"
   },
+  "explanation_text": {
+    "tags_helper_aria_label": "Show tags explanation",
+    "tags_helper_text": "Tags describes the contents of a resource",
+    "tags_helper_text_edit_resource": "Quality tags increases the chance that others will find and use your resource. You can readily add more than one tag if desired. It is possible to add or remove tags after publication of your resource",
+    "tags_helper_text_example": "For instance can subjects, themes, and field of study be used."
+  },
```

Comment:
Gjerne se over at teksten ser grei ut.

Reply:
Jeg likte at du hadde et konkret eksempel - men jeg synes bare det ble litt teknisk. Kanskje ha noe mange kan kjenne seg igjen i ?

Synes fremdeles bare vi kan droppe "readily". Evt slå sammen og skrive: "Add one or more tags, to increase the chance...."

Reply:
Ser det bedre ut nå?